modules/local/lookup_kingdom.nf

```diff
@@ -0,0 +1,26 @@
+process LOOKUP_KINGDOM {
+
+    tag "${meta.prefix}"
+
+    label 'process_light'
+
+    container 'quay.io/microbiome-informatics/genomes-pipeline.python3base:v1.1'
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), file("${meta.prefix}_kingdom.txt"), emit: detected_kingdom
```

Comment:
```suggestion
    tuple val(meta), path("${meta.prefix}_kingdom.txt"), emit: detected_kingdom
```
`path` is preferred to `file` https://www.nextflow.io/docs/latest/process.html#input-type-file